cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -1314,6 +1318,10 @@ export default {
     );
   },
 
+  clickItemBarcodeLink(barcode) {
```

Comment:
We already have the function for opening item view InventoryInstance.openHoldingItem();